Microsoft.Toolkit.Uwp.UI.Controls/TabView/TabView.xaml

```diff
@@ -0,0 +1,1034 @@
+                    </Viewbox>
+                </Button>
+            </Grid>
+            <ContentControl Content="{Binding (ex:FrameworkElementExtensions.Ancestor).TabActionHeader, ElementName=TabActionHeaderContainer}" />
+        </StackPanel>
+    </DataTemplate>
+
+    <Thickness x:Key="TabViewItemHeaderIconMargin">12,0,0,0</Thickness>
+    <Thickness x:Key="TabViewItemHeaderCloseMargin">-12,0,0,0</Thickness>
+    <x:Double x:Key="TabViewItemHeaderIconSize">16</x:Double>
+
+    <Style x:Key="BaseTabViewStyle"
+           TargetType="local:TabView">
+        <Setter Property="Background" Value="{ThemeResource SystemControlBackgroundChromeMediumLowBrush}" />
+        <Setter Property="VerticalAlignment" Value="Stretch" />
+        <Setter Property="HorizontalAlignment" Value="Stretch" />
+        <Setter Property="IsTabStop" Value="False" />
+        <Setter Property="TabNavigation" Value="Once" />
+        <Setter Property="IsSwipeEnabled" Value="True" />
+        <Setter Property="ScrollViewer.HorizontalScrollBarVisibility" Value="Hidden" />
+        <Setter Property="ScrollViewer.VerticalScrollBarVisibility" Value="Disabled" />
+        <Setter Property="ScrollViewer.HorizontalScrollMode" Value="Enabled" />
+        <Setter Property="ScrollViewer.IsHorizontalRailEnabled" Value="False" />
+        <Setter Property="ScrollViewer.VerticalScrollMode" Value="Disabled" />
+        <Setter Property="ScrollViewer.IsVerticalRailEnabled" Value="False" />
+        <Setter Property="ScrollViewer.ZoomMode" Value="Disabled" />
+        <Setter Property="ScrollViewer.IsDeferredScrollingEnabled" Value="False" />
+        <Setter Property="ScrollViewer.BringIntoViewOnFocusChange" Value="True" />
+        <Setter Property="TabActionHeaderTemplate" Value="{StaticResource DefaultTabActionHeaderTemplate}" />
+        <Setter Property="ItemHeaderTemplate" Value="{StaticResource DefaultItemHeaderTemplate}" />
+        <Setter Property="ItemContainerStyle" Value="{StaticResource DefaultTabViewItemStyle}" />
+        <Setter Property="ItemContainerTransitions">
+            <Setter.Value>
+                <TransitionCollection>
+                    <AddDeleteThemeTransition />
+                    <ContentThemeTransition />
+                    <EntranceThemeTransition IsStaggeringEnabled="False" />
+                </TransitionCollection>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="ItemsPanel">
+            <Setter.Value>
+                <ItemsPanelTemplate>
+                    <StackPanel Orientation="Horizontal" />
+                </ItemsPanelTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Template" Value="{StaticResource TabViewTemplate}" />
+    </Style>
+
+    <!--  Default Compact/WPF Style  -->
+    <Style BasedOn="{StaticResource BaseTabViewStyle}"
+           TargetType="local:TabView" />
+
+    <Style x:Key="TabViewDocumentStyle"
```

Comment:
This style is unnecessary, could be just a documentation example

Reply:
I don't think we should be shipping separate styles - in most cases styles have to be updated anyway to get what you want. I'd rather have one style and document common scenarios. Especially since the style only changes default values